src/Yaapii.Atoms/Map/MapOf.cs

```diff
@@ -135,11 +135,8 @@ public MapOf(IDictionary<string, string> entries) : base(() => entries, false)
         /// <summary>
         /// A map from string to string.
         /// </summary>
-        public MapOf(string key, string value, params string[] additional) : this(
```

Comment:
This will introduce a breaking change for devs using this ctor. I think we need both ctors to not break other devs code again.

Reply:
It will still break because the ctors then are ambiguous.